Makefile

```diff
@@ -1,5 +1,6 @@
+
 MOD_FLAGS := $(shell (go version | grep -q -E "1\.(11|12)") && echo -mod=vendor)
-CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | xargs -I{} basename {}))
+CMDS  := $(addprefix bin/, $(shell go list $(MOD_FLAGS) ./cmd/... | grep -Fv "github.com/operator-framework/operator-registry/cmd/cli" |xargs -I{} basename {}))
```

Comment:
If we keep the command declarations in separate top level packages, we don't need to do this right?

Reply:
You mean put this in `pkg` instead of `cmd`?

Reply:
I mean `cmd/initializer`, `cmd/registry`, `cmd/opm`, and forgo extracting the commands out to `cmd/cli` -- but it seems like everyone has a different opinion on this, so feel free to ignore.